docs/product-assembly_pam-on-openshift/src/main/asciidoc/environment-authoring-ha-proc.adoc

```diff
@@ -0,0 +1,42 @@
+[id='environment-authoring-ha-proc']
+= Deploying a high-availability authoring environment
+
+To deploy a high-availability authoring environment, use the `rhpam-70-authoring-ha.yaml` template file. You can download this file from `{URL_BASE_GITHUB_PAM}/templates`.
+
+.Procedure
+. Use one of the following methods to deploy the template:
+.. In the OpenShift Web UI, select *Add to Project >  Import YAML / JSON* and then select or paste the `rhpam-70-authoring-ha.yaml` file. In the *Add Template* window, ensure *Process the template* is selected and click *Continue*.
+.. To use the OpenShift command line console, prepare the following command line:
++
+[subs="verbatim,macros"]
+----
+oc new-app -f <template-path>/rhpam-70-authoring-ha.yaml -p PARAMETER=value -p PARAMETER=value...
+----
++
+In this command line:
++
+* Replace `<project-name>` with the name of your project
+* Replace `<template-path>` with the path to the downloaded template file.
+* Use as many `ATTRIBUTE=value` pairs as needed to set the required attributes. You can view the template file to see descriptions for the attributes.
++
+. Set the following mandatory attributes:
+** *Application Name* (`APPLICATION_NAME`): the name of the OpenShift application. It is used in the default URLs for {CENTRAL} and {KIE_SERVER}. OpenShift also uses the application name to create a separate set of deployment configurations, services, routes, labels, and artifacts. You can deploy several applications using the same template into the same project, as long as you use different application names. 
```

Comment:
I notice throughout this doc and in most of the other docs that you write (I've mentioned it in previous peer reviews) that you tend toward the format "Some text before a colon: text after a colon". Our style per RH and IBM SG is for capitalization after the colon, "Some text before a colon: Text after a colon." At least in situations like this. The only time it's not capped is in-line, such as "This option configures three elements: dependencies, repositories, and plugins."

Trivial, but just letting you know.

Reply:
A peculiarity of the English language helped me out here. I just replaced ": t" with ": T" and it covered basically all of them

Reply:
(though I hag to fix main.adoc :) )